arduino/obstacleAvoidance/usedpins.h

```diff
@@ -0,0 +1,24 @@
+// header-file defining the aruduino pins used 
+// for the different sensors and actuators
+
+#ifndef USEDPINS_H
+#define USEDPINS_H
+
+// left motor pins
+#define BRUSHED_LEFT_FORWARD_PIN	8
+#define BRUSHED_LEFT_BACKWARD_PIN	10
+#define BRUSHED_LEFT_ENABLE_PIN		9
```

Comment:
Why are you using macros to declare simple constants? Isn't `const int` or, if you want to go really low on memory, `const uint8_t` (aka `const unsigned char`) good enough for you? :cold_sweat:
Read more: [ES.31: Don't use macros for constants or "functions"](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-macros2).

Reply:
Changed all macros to constants in latest commit.